packages/odyssey-react/src/components/Table/TableSortButton.module.scss

```diff
@@ -31,21 +31,21 @@
   inset-block-start: 50%;
   inset-inline-end: 0;
   margin-block: 0;
-  margin-inline-start: var(--SpaceMarginInlineStartIndicator);
+  margin-inline-start: var(--SortIndicatorMarginInlineStart);
   margin-inline-end: 0;
   transform: translateY(-50%);
-  color: var(--ColorIndicator);
-  font-size: var(--SizeIndicator);
+  color: var(--SortIndicatorTextColor);
+  font-size: var(--SortIndicatorFontSize);
 }
 
 .unsortedDirection {
   .sortIndicator {
-    color: var(--ColorIndicatorUnsorted);
+    color: var(--UnsortedSortIndicatorTextColor);
   }
 
   &:hover {
     .sortIndicator {
-      color: var(--ColorIndicatorUnsortedHover);
+      color: var(--UnsortedHoverSortIndicatorTextColor);
```

Comment:
oh boy, this one is fun.

To me, SortIndicator should come before Hover as it's a child part and Hover is the state?

Reply:
👍 addressed in edef7d7